src/docs/asciidoc/user-guide/23-tasks.adoc

```diff
@@ -6,7 +6,7 @@ The plugin provides a set of tasks for your project and preconfigures them with
 |=======
 |Task name                 |Depends On                |Type                                                                                 |Description
 |`dockerSyncBuildContext`  |`classes`                 |TaskProvider<{uri-gradle-docs}/javadoc/org/gradle/api/tasks/Sync.html[Sync]>                       |Copies the application files to a temporary directory for image creation.
-|`dockerCreateDockerfile`  |`dockerSyncBuildContext`       |TaskProvider<{uri-ghpages}/api/com/bmuschko/gradle/docker/tasks/image/Dockerfile.html[Dockerfile]> |Creates the Docker image for the Java application.
+|`dockerCreateDockerfile`  |`dockerSyncBuildContext`       |TaskProvider<{uri-ghpages}/api/com/bmuschko/gradle/docker/tasks/image/Dockerfile.html[Dockerfile]> |Creates the `Dockerfile` for the Java application.
```

Comment:
Can you make the same change in `33-tasks.adoc`?

Reply:
Sure, commit amended and force-pushed.